docker/run/Dockerfile

```diff
@@ -43,14 +44,19 @@ ENV \
     SBT_VERSION=0.13.13-2 \
     YARN_VERSION=1.17.3 \
     # Scanner versions.
-    SCANCODE_VERSION=2.9.7
+    SCANCODE_VERSION=2.9.7 \
+    # Installation directories.
+    FLUTTER_HOME=/opt/flutter
+
+ENV PATH="$PATH:$FLUTTER_HOME/bin:$FLUTTER_HOME/bin/cache/dart-sdk/bin"
```

Comment:
Does it work to add this to the preceding `ENV` block so we do not introduce another layer, or is `FLUTTER_HOME` then not yet expanded correctly?

Reply:
We have it like this in another Docker container, but I did not try if it would also work in one `ENV` block.

Reply:
Ok, then I guess it has to be like that, as I vaguely remember to have tried it once.